src/blueapi/cli/cli.py

```diff
@@ -22,29 +22,15 @@ def main(ctx) -> None:
         print("Please invoke subcommand!")
 
 
-@main.command(name="worker")
-@click.option("-c", "--config", type=Path, help="Path to configuration YAML file")
-def start_worker(config: Optional[Path]):
-    from blueapi.service import start
+@click.version_option(version=__version__)
+@main.command(name="run")
```

Comment:
Was a reason for changing the name of this command by the way?

Reply:
yeah, I preferred it. `bluesky worker` made sense when you were actually spinning up a service that started the worker. But here, this command is starting a fastAPI application, which indirectly (through startup tasks) starts up a worker.

Reply:
yeah, I don't know what you prefer @callumforrester I'm happy to keep it 'worker' it just seems out of style with other cli tools?

Reply:
So the problem is that the command `run` goes into the global namespace, so in your terminal you can type `ls`, `rm`, `cat` or now `run`. It definitely doesn't belong there, it needs to be called something that indicates it has anything to do with Bluesky for a start.